common/pulp_rpm/common/constants.py

```diff
@@ -118,6 +118,7 @@
 CONFIG_SSL_AUTH_CA_CERT = 'ssl_auth_ca_cert'
 
 # Copy operation config
+CONFIG_ADDITIONAL_REPOS = 'additional_repos'
```

Comment:
we need to add this option to technical reference docs https://docs.pulpproject.org/plugins/pulp_rpm/tech-reference/yum-plugins.html#yum-importer